compiler-rt/lib/rtsan/rtsan_context.cpp

```diff
@@ -85,12 +89,13 @@ bool __rtsan::Context::InRealtimeContext() const { return realtime_depth_ > 0; }
 
 bool __rtsan::Context::IsBypassed() const { return bypass_depth_ > 0; }
 
-void __rtsan::PrintDiagnostics(const char *intercepted_function_name) {
+void __rtsan::PrintDiagnostics(const char *intercepted_function_name, uptr pc,
```

Comment:
Would it be possible to get more descriptive names than `pc` and `bp`?

Reply:
The full names of these are "program counter" and "base pointer". We can use their full names, but it would be breaking with the rest of the code,

```
> rg "ProgramCounter|program_counter|programCounter"| wc -l
      11
> rg "uptr pc" | wc -l
     428
```

(The comparison with base pointer is a little harder to illustrate with a bulk search for this as the term is a little overloaded.)

Reply:
OK cool, if that's the style throughout the rest of the codebase I think it's important to be consistent with that. Thanks!